docs/manual/optional-checker.tex

```diff
@@ -98,6 +98,36 @@
 \label{fig-optional-hierarchy}
 \end{figure}
 
+\subsectionAndLabel{Optional method annotations}{optional-method-annotations}
+
+The Optional Checker supports several annotations that specify method
+behavior.  These are declaration annotations, not type annotations:  they
+apply to the method itself rather than to some particular type.
+
+\begin{description}
+
+\item[\refqualclass{checker/optional/qual}{RequiresPresent}]
+  indicates a method precondition:  The annotated method expects the
+  specified variables to be a present Optional when this
+  method is invoked.  Don't use this for formal parameters (just annotate
```

Comment:
This annotation can't be applied to parameters, so this sentence doesn't make sense.  Unless you mean that the expression in the `@RequiresPresent` should not be a parameter. If so, please reword.